### PR TITLE
add galleries with react-slick carousel

### PR DESCRIPTION
**Pull request by Joanartdacosta**
add react-slick 
add carousel component
add several arrays (galleries)

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -11,8 +11,11 @@
     "next": "13.4.10",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-router-hash-link": "^2.4.3",
```
Comment (doublethink13) [inline app/package.json:14]: please use fixed versions

File: app/package.json [hunk 2]
```diff
@@ -11,8 +11,11 @@
...
+    "react-slick": "^0.29.0",
     "remark": "14.0.3",
-    "remark-html": "15.0.2"
+    "remark-html": "15.0.2",
+    "slick-carousel": "^1.8.1"
```
Comment (doublethink13) [inline app/package.json:18]: verison

File: app/package.json (hunk 1 above)
Comment (Joanartdacosta) [reply app/package.json]: done!

File: app/package.json (hunk 2 above)
Comment (Joanartdacosta) [reply app/package.json]: done!

**Pull request merged**